ros2_batch_job/__main__.py

```diff
@@ -535,7 +535,10 @@ def run(args, build_function, blacklisted_package_names=None):
 
         # Fetch colcon mixins
         if args.colcon_mixin_url:
-            job.run([args.colcon_script, 'mixin', 'remove', 'default', '||', 'true'], shell=True)
+            if sys.platform == 'win32':
+                job.run([args.colcon_script, 'mixin', 'remove', 'default', '||', 'VER>NUL'], shell=True)
+            else:
+                job.run([args.colcon_script, 'mixin', 'remove', 'default', '||', 'true'], shell=True)
```

Comment:
I would suggest to avoid the duplication and set a variable in the two conditional blocks and use that in the `job.run()` call as one arguments.

Reply:
a109630